plugins/aggregators/histogram/histogram.go

```diff
@@ -63,6 +69,8 @@ type groupedByCountFields struct {
 	fieldsWithCount map[string]int64
 }
 
+var timeNow timeFunc
```

Comment:
Does this also work?
```suggestion
var timeNow func() time.Time = time.Now
```
It saves two lines.